spec/change_set_persisters/plum_change_set_persister_spec.rb

```diff
@@ -623,4 +623,55 @@
       expect(solr_record["member_of_ssim"]).to eq ["id-#{parent.id}"]
     end
   end
+
+  context "setting visibility from remote metadata" do
```

Comment:
Add a test for the case where it's not checked?

Reply:
Added.